### PR TITLE
chore(upgrade): migrate from CRA to Vite

### PR DESCRIPTION
**Pull request by oyo**
## Description

We migrated to the Vite Framework as Create React App is deprecated.

## Why

We were unable to upgrade dependencies for a while already and migrated to a state of the art build framework.

## Issue

#123 

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [ ] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [x] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/cax-responsibilities.tsx [hunk 1]
```diff
@@ -90,15 +96,14 @@
...
   const handleSendInvite = () => {
-    setInvitedResponse({severity: SeverityType.ERROR, message: ''})
+    setInvitedResponse({ severity: SeverityType.ERROR, message: '' })
     if (email && validateEmail(email)) {
       setLoading(true)
       const user = {
-        email: email,
-        roles: [role],
-        message: message,
+        email: email ?? '',
```
[the remark this answers is not in the document]
Comment (oyo) [reply src/components/cax-responsibilities.tsx]: fixed

**Comment by evegufy** [comment]
IP issues created:
A review is required for npm/npmjs/-/react-datepicker/6.8.0.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14379
A review is required for npm/npmjs/-/qs/6.12.1.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14380
A review is required for npm/npmjs/-/dedent/1.5.3.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14381
A review is required for npm/npmjs/-/react-bootstrap/2.10.2.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14382
A review is required for npm/npmjs/-/i18next/23.11.2.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14383
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14384
A review is required for npm/npmjs/@testing-library/react/14.3.1.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14385
A review is required for npm/npmjs/-/react-icons/5.1.0.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14386
A review is required for npm/npmjs/@typescript-eslint/eslint-plugin/7.7.0.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14387
A review is required for npm/npmjs/-/react-player/2.16.0.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14388
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14389
A review is required for npm/npmjs/@typescript-eslint/typescript-estree/7.7.0.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14390

**Review by evegufy — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -1,95 +1,14 @@
 {
-  "name": "cx-portal-registration",
-  "version": "v1.7.0-RC1",
+  "name": "@catena-x/portal-frontend-registration",
+  "version": "v1.6.0",
```
[the remark this answers is not in the document]
Comment (oyo) [reply package.json]: Done - bumped to v2.0.0-RC1

File: package.json [hunk 3]
```diff
@@ -102,5 +21,102 @@
...
+    "@react-hook/cache": "^1.1.1",
+    "@reduxjs/toolkit": "^2.2.1",
+    "axios": "^1.6.8",
+    "bootstrap": "^5.3.3",
+    "buffer": "^6.0.3",
+    "dayjs": "^1.11.10",
+    "history": "^5.3.0",
+    "i18next": "^23.10.1",
+    "i18next-browser-languagedetector": "^7.2.0",
+    "keycloak-js": "^24.0.2",
```
[the remark this answers is not in the document]
Comment (oyo) [reply package.json]: Done

**Comment by evegufy** [comment]
:warning: 

The following IP issues ran into a manual review by the EF IP team:

- npm/npmjs/-/dedent/1.5.3. https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14381
- npm/npmjs/-/react-bootstrap/2.10.2. https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/14382

The review will take a couple days. As those dependencies will most certainly be approved and to be able to test the change on dev env, the PR is merged to the dev branch with those "restricted" dependencies to the dev branch. We'll keep monitoring the IP issues and no release will occur with restricted in the DEPENDENCIES file.

**Review by evegufy — APPROVED** [review]

**Pull request merged**